site/content/docs/user/quick-start.md

```diff
@@ -169,6 +173,37 @@ nodes:
   replicas: 2
 ```
 
+kind also supports multiple control-plane and HA setups using a stacked etcd topology:
+```yaml
+# six node cluster with three control-plane and three worker nodes
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+  replicas: 3
+- role: worker
+  replicas: 3
+```
+
+Next to `role` and `replicas` you can also specify fields like `image` to be able to
+use a custom image per node:
+```yaml
+# a setup with custom control-plane and worker node images
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+  image: mycontrolplane-node-image:latest
+- role: worker
+  image: myworker-node-image:latest
```

Comment:
we should put a concrete upstream kind image for each of these (EG the current default for one of them)

Reply:
we can do that, except that someone has to update this document on each release.

Reply:
It's OK if these get old, but it sends a better signal than suggesting totally custom images currently.
We also need to update these to the current API version each release ideally anyhow until we get out of alpha APIs.